app/modules/asset_groups/models.py

```diff
@@ -119,6 +119,98 @@ class AssetGroupSighting(db.Model, HoustonModel):
     # May have multiple jobs outstanding, store as Json obj uuid_str is key, In_progress Bool is value
     jobs = db.Column(db.JSON, nullable=True)
 
+    def commit(self):
+        from app.modules.utils import Cleanup
+        from app.modules.sightings.models import Sighting
+        from app.modules.encounters.models import Encounter
+
+        if self.stage != AssetGroupSightingStage.curation:
+            raise HoustonException(
+                message=f'AssetGroupSighting {self.guid} is currently {self.stage}, not curating cannot commit',
+            )
+
+        if not self.config:
+            raise HoustonException(
+                message=f'AssetGroupSighting {self.guid} has no metadata',
+            )
+
+        # Create sighting in EDM
+        result_data = current_app.edm.request_passthrough_result(
```

Comment:
this has the potential to have edm complain (non-200 response) in a lot of different ways, if the metadata is wrong -- things like invalid values for certain fields, missing requirements, etc.... wonder how best this can abort in a way to notify frontend as to _why_ the data was rejected?

Reply:
Interesting point. This is what was previuosly in AssetGroup resources.py and copied and tweaked from sightings. 
Certainly something with scope for improvement in future

Reply:
probably try/except.  but this might be enough for now and we can play "wait and see".  there will be a lot of data-validation going on edm.  generally (when things were non-pipeline straight sightings POST) this was expressed back to frontend via an `errorFields: [ ... ]` parameter in the response -- a list of the fields that were (somehow) wrong.